cmd/saml2aws/commands/login.go

```diff
@@ -31,6 +32,8 @@ func Login(loginFlags *flags.LoginExecFlags) error {
 	}
 
 	sharedCreds := awsconfig.NewSharedCredentials(account.Profile, account.CredentialsFile)
+	// creates a cacheProvider, only used when --cache is set
+	cacheProvider := samlcache.NewSAMLCacheProvider(account.Name, "")
 
```

Comment:
Is the second argument to `NewSAMLCacheProvider` always an empty string?

Reply:
that's right;
It's mimicking `NewSharedCredentials` and is a placeholder that will allow parameterising the file location in the future. I thought it would be too heavy to bundle this in this PR and not sure how the cli would look like either.
At the moment it's empty string, which means we'll always use the default location.

I'm happy to remove the extra parameter if it adds confusion.

Reply:
Considered what you said and simplified the SAMLCacheProvider creation altogether. We can add the filename later if we want to as easily.